include/crypto/evp.h

```diff
@@ -230,6 +230,10 @@ struct evp_kdf_st {
     OSSL_FUNC_kdf_set_ctx_params_fn *set_ctx_params;
 };
 
+#define EVP_ORIG_DYNAMIC    0
+#define EVP_ORIG_GLOBAL     1
+#define EVP_ORIG_METH       2
```

Comment:
Since this is internal, it should be perfectly safe to make this an enum, if you prefer.

Reply:
> Since this is internal, it should be perfectly safe to make this an enum, if you prefer.

I prefer to just follow the crowd. :)